core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -7,8 +7,15 @@
       <img class="conversation-skin-oppia-avatar"
            ng-src="<[OPPIA_AVATAR_IMAGE_URL]>" alt="">
       <div class="conversation-skin-tutor-card-top-content protractor-test-conversation-content"
-           angular-html-bind="activeCard.contentHtml"
            focus-on="<[getContentFocusLabel($index)]>">
+        <span angular-html-bind="activeCard.contentHtml">
```

Comment:
Not div? Content may span multiple paragraphs, for instance. Might mean the next bit has to be a div too.